addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -236,84 +262,115 @@ String Netatmo_LivingRoom_HomeCoach_HealthIndex "Climate" { channel = "netatmo:N
+| welcomeEventType        | String    | Type of event                                            |
+| welcomeEventTime        | DateTime  | Time of occurrence of event                               |
+| welcomeEventCameraId    | String    | Camera that detected the event                           |
+| welcomeEventPersonId    | String    | Id of the person the event is about (if any)             |
+| welcomeEventSnapshot    | Image     | picture of the last event, if it applies                 |
+| welcomeEventSnapshotURL | String    | if the last event (depending upon event type) in the home lead a a snapshot picture, the picture URL will be available here |
+| welcomeEventVideoURL    | String    | if the last event (depending upon event type) in the home lead a a snapshot picture, the corresponding video URL will be available here |
+| welcomeEventVideoStatus | String    | Status of the video (recording, deleted or available)    |
+| welcomeEventIsArrival   | Switch    | If person was considered "away" before being seen during this event |
+| welcomeEventMessage     | String    | Message sent by Netatmo corresponding to given event     |
+| welcomeEventSubType     | String    | Sub-type of SD and Alim events                           |
+
+All these channels are read only.
 
 
 ### Welcome Camera
 
 **Supported channels for the Camera thing:**
 
-* welcomeCameraStatus
-* welcomeCameraSdStatus
-* welcomeCameraAlimStatus
-* welcomeCameraIsLocal : indicates wether the camera is on the same network than the openHab Netatmo Binding
-* welcomeCameraLivePicture : current image snapshot
-* welcomeCameraLivePictureUrl : url of the current image
-* welcomeCameraLiveStreamUrl : url of the feed for live video
+| Channel ID                  | Item Type | Description                                              |
+|-----------------------------|-----------|----------------------------------------------------------|
+| welcomeCameraStatus         | Switch    | State of the camera                                      |
+| welcomeCameraSdStatus       | Switch    | State of the SD card                                     |
+| welcomeCameraAlimStatus     | Switch    | State of the power connector                             |
+| welcomeCameraIsLocal        | Switch    | indicates whether the camera is on the same network than the openHab Netatmo Binding |
+| welcomeCameraLivePicture    | Image     | Camera Live Snapshot                                     |
+| welcomeCameraLivePictureUrl | String    | Url of the live snapshot for this camera                 |
+| welcomeCameraLiveStreamUrl  | String    | Url of the live stream for this camera                   |
+
+All these channels are read only.
+
+Warning : the URL of the live snapshot is a fixed URL so the value of the channel welcomeCameraLivePictureUrl will never be updated once first set by the binding. So to get a refreshed picture, you need to use the refresh parameter in your sitemap image element. 
```

Comment:
Linter error: new line after full stop.

Reply:
Sorry, I don't understand what I should update.

Reply:
To facilitate reviews and diffs, there should be a new line for every sentence within a paragraph. (See 
https://www.eclipse.org/smarthome/documentation/development/bindings/docs.html)